### PR TITLE
Review README

### PR DESCRIPTION
**Pull request by cedricosei**
This update contains an initial review of the README files, which describe the main and subdirectories. In addition it contains a small adjustment in the license file (adding of year and names).

### CODE REVIEW

File: data/README.md
Comment (jf87) [inline data/README.md:1]: Please check that the paths are correct. The first two ones are already not correct

**Comment by jf87** [comment]
Now the overall README and the data and src README.md files contain duplicated info. Please fix.

File: data/README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# Dataset
+# Data Directory
 
+The following directory contains a variety of data used in the SM3-Text-to-Query project. 
+
+Below you will find a brief description of the specific subdirectories.
+
+
+- `./synthea_database` contains the Synthea data used as the basis for PostgreSQL, Graph DB (RDF), Neo4j, and MongoDB.
```
Comment (jf87) [inline data/README.md]: The paths are still wrong. Please check all things carefully before submitting it...

**Comment by cedricosei** [comment]
I updated the paths and changed the project README slightly such that we don't have the duplicate definition as mentioned above. I made sure to double check it more thoroughly this time.

File: README.md [hunk 2]
```diff
@@ -24,27 +24,15 @@ This work will be presented at [NeurIPS 2024](https://neurips.cc/virtual/2024/po
 **Note: This repository will be updated further during the next week to make it easier to set up and use the four different databases.**
 
 ## Data
+The `./data` directory contains various used templates, synthea data, train and dev data and the compoted results.
```
Comment (jf87) [inline README.md]: Fix to:
The `./data` directory contains our template questions, the Synthea data, train and dev data, and the Text-to-Query results of our evaluated LLMs for all databases.

File: README.md [hunk 3]
```diff
@@ -24,27 +24,15 @@ This work will be presented at [NeurIPS 2024](https://neurips.cc/virtual/2024/po
...
-  MQL
-- `./data/dataset/processed_data` contains different data splits, including the responses from the respective databases.
-- `./data/dataset/question_templates` contains the 408 question templates used to construct SM3-Text-to-Query.
-- `./data/results` contains the results for each of the
-  evaluated database systems and query languages for each of the five prompts,
-the four LLMs for dev (results on test are not published to not expose the questions)
+For detailed information about the data presented in this project, please refer to the README in the `./data` directory. 
 
 ## Code
+The `./src` directory contains the code to reproduce the results presented in SM3-Text-to-Query research.
```
Comment (jf87) [inline README.md]: The `./src` directory contains the code to reproduce the results presented in SM3-Text-to-Query.

**Review by jf87 — CHANGES_REQUESTED** [review]
see comments

**Comment by cedricosei** [comment]
Change requests were implemented as described in the comments above.

**Review by jf87 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**